models/staging/stripe/stg_payments.sql

```diff
@@ -4,7 +4,7 @@ with payments as (
         ORDERID as order_id,
         PAYMENTMETHOD as payment_method,
         STATUS,
-        AMOUNT,
+        {{ cents_to_dollars('amount', 4) }} as AMOUNT,

```

Comment:
You will want to keep this as just `{{ cents_to_dollars('AMOUNT') }}` because generally, you want only 2 decimal places for dollars.

Reply:
great, thanks!  made the suggested edit.